File: shanhaijing-travel/replace.js
const fs = require('fs');
const file = 'c:/Users/hp/OneDrive/Desktop/Full HACK/shanhaijing-travel/index.html';
let content = fs.readFileSync(file, 'utf-8');
content = content.replace(/<!-- 中国矢量地图SVG[\s\S]*?<template id="china-map-svg-template">[\s\S]*?<\/template>/, '');
fs.writeFileSync(file, content, 'utf-8');

File: shanhaizhi/replace.py
import re

file_path = r'c:\Users\hp\OneDrive\Desktop\Full HACK\shanhaijing-travel\index.html'

with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

# Find everything from <template id="china-map-svg-template"> to </template>
new_content = re.sub(r'<!-- 中国矢量地图SVG.*?<template id="china-map-svg-template">.*?</template>', '', content, flags=re.DOTALL)
if new_content == content:
    # try just the template tag
    new_content = re.sub(r'<template id="china-map-svg-template">.*?</template>', '', content, flags=re.DOTALL)

with open(file_path, 'w', encoding='utf-8') as f:
    f.write(new_content)

print("Done. Matched:", content != new_content)
